2013/data_file_specification.json

```diff
@@ -260,6 +390,9 @@
                "end": "54",
                "length": "5",
                "dataType": "N",
+               "format": "currency,thousands",
```

Comment:
I realized after @LinuxBozo agreed with me that my datetime example didn't accurately reflect the input value I was trying to match, but was instead indicating a possible output format for a datetime value.

What I should have said was, maybe the datetime item for the transmittal sheet's timestamp could be:

```json
format: {
    type: 'datetime',
    match: 'yyyyMMddHHmm' 
}
```